named_arrays/_functions/functions.py

```diff
@@ -45,6 +45,57 @@ def outputs(self) -> na.AbstractArray:
 
     __named_array_priority__: ClassVar[float] = 100 * na.AbstractVectorArray.__named_array_priority__
 
+    @property
+    def axes_center(self) -> tuple(str):
+        """
+        Return keys corresponding to all input axes representing bin centers
+        """
+        axes_center = tuple()
+        input_shape = self.inputs.shape
+        output_shape = self.outputs.shape
+        all_axes = set(input_shape | output_shape)
```

Comment:
I think it might be safer to use `na.broadcast_shapes()` here.

Reply:
I think that suggestion doesn't work when two shapes a shared axis with different length, as is the case with a vertex axes.